docs/src/development/troubleshoot.md

```diff
@@ -96,6 +96,15 @@ platform project:clear-build-cache -p <PROJECT_ID>
 
 The next build for each environment is likely to take longer as the cache rebuilds.
 
+### Access the project
```

Comment:
This is a different kind of thing than the above. The above are tasks that you actively want to do. This is about an error you might run into while doing a common task. I would not keep these in the same structure.

I'm not certain this extra section is required, but it should at least be in a different section.

Reply:
In my latest commit, I moved the headline one level up. Would that be enough?
Or should that part be re-located somewhere else?